### PR TITLE
fixing the bug in Mat Dev.

### PR DESCRIPTION
**Pull request by RohitKandimalla**

### CODE REVIEW

File: src/main/java/mat/server/util/MeasureUtility.java [hunk 1]
```diff
@@ -69,6 +69,15 @@ public static String formatVersionText(String revisionNumber, String version) {
         return formatVersionText(version) + "." + revisionFormat.format(Integer.parseInt(revisionNumber));
     }
 
+    public static String formatVersionTextForIncludeLibraries(String version) {
+        StringUtility su = new StringUtility();
+        String[] versionArr = version.split("\\.");
```
Comment (serhii-ilin) [inline src/main/java/mat/server/util/MeasureUtility.java]: should we handle safely cases when a version value has less than 3 or even 0 elements?

Comment (RohitKandimalla) [reply src/main/java/mat/server/util/MeasureUtility.java]: Yes, working on it.

File: src/main/java/mat/server/util/MeasureUtility.java [hunk 2]
```diff
@@ -69,6 +69,22 @@ public static String formatVersionText(String revisionNumber, String version) {
         return formatVersionText(version) + "." + revisionFormat.format(Integer.parseInt(revisionNumber));
     }
 
+    public static String formatVersionTextForIncludeLibraries(String version) {
+        StringUtility su = new StringUtility();
+        String[] versionArr = version.split("\\.");
+        String majorVersion = "", minorVersion = "", revisionNumber = "";
```
Comment (serhii-ilin) [inline src/main/java/mat/server/util/MeasureUtility.java]: I believe most of the code style checkers recommend to define each variable on a separate line.

File: src/main/java/mat/server/util/MeasureUtility.java [hunk 3]
```diff
@@ -69,6 +69,22 @@ public static String formatVersionText(String revisionNumber, String version) {
...
+        for(int i = 0; i < versionArr.length; i++) {
+            if(i == 0) {
+                majorVersion = su.trimLeadingZeros(versionArr[0]);
+            } else if(i == 1) {
+                minorVersion = su.trimLeadingZeros(versionArr[1]);
+            } else {
+                revisionNumber = su.trimLeadingZeros(versionArr[2]);
+            }
+        }
+        return majorVersion + "." + minorVersion + "." + revisionFormat.format(Integer.parseInt(revisionNumber == "" ? "000" : revisionNumber));
```
Comment (serhii-ilin) [inline src/main/java/mat/server/util/MeasureUtility.java]: so, there is a possibility to see "..." version, interesting..

File: src/main/java/mat/server/util/MeasureUtility.java (hunk 2 above)
Comment (RohitKandimalla) [reply src/main/java/mat/server/util/MeasureUtility.java]: Updated it

**Pull request merged**